client/verta/verta/deployment/resources.py

```diff
@@ -36,6 +36,13 @@ class CpuMillis(_Resource):
     """
     Number of CPU milli allowed for Endpoint. Must be an integer greater than 0.
 
+    Examples
+    --------
+    .. code-block:: python
+
+        from verta.deployment.resources import CpuMillis
+        cpu = CpuMillis(128)
```

Comment:
1000 millis = 1 CPU core, so a nice fraction of 1000 like `250` makes more sense than `128`.
```suggestion
        cpu = CpuMillis(250)
```

Reply:
Fixed.